clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -185,9 +215,29 @@ public CompletableFuture start() throws Exception {
             return transport.send(handshake).thenRun(() -> {
                 hubConnectionStateLock.lock();
                 try {
+                    resetServerTimeout();
```

Comment:
I don't believe we should be resetting the server timeout here since we aren't receiving a message